src/test/kotlin/uk/gov/dluhc/notificationsapi/mapper/RejectedDocumentsMapperTest.kt

```diff
@@ -62,13 +68,28 @@ class RejectedDocumentsMapperTest {
         // Then
         assertThat(actual).usingRecursiveComparison().isEqualTo(
             listOf(
-                "Utility Bill - Document is too old, Document is not readable",
-                "Birth Certificate - Document is not readable",
-                "Mortgage statement - other - Some notes",
-                "Firearms cert - More notes",
+                "Utility Bill\n" +
```

Comment:
Do we need to worry about VAC changes to letters/emails?

Reply:
Will double check once merged, but as far as I can tell VAC uses a different set of notification API endpoints and mappers (called "ID document required" and "ID document resubmission"). OAVA uses "NINO not matched" and "Rejected document" in place of them respectively.

Possibly some duplication going on that can be eliminated but not looked into it